tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./index.html",
+    "./src/**/*.{js,ts,jsx,tsx}",
```

Comment:
no need to watch all files, it's sufficient to watch only for tsx files since that's what you'll be writing
if you also plan on adding a lib folder you need to watch for styles in its components directory
```suggestion
    "./src/**/*.tsx",
```